src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -517,18 +517,19 @@ public function makePathRelative($endPath, $startPath)
      *  - existing files in the target directory will be overwritten, except if they are newer (see the `override` option)
      *  - files in the target directory that do not exist in the source directory will not be deleted (see the `delete` option)
      *
-     * @param string            $originDir The origin directory
-     * @param string            $targetDir The target directory
-     * @param \Traversable|null $iterator  Iterator that filters which files and directories to copy, if null a recursive iterator is created
-     * @param array             $options   An array of boolean options
-     *                                     Valid options are:
-     *                                     - $options['override'] If true, target files newer than origin files are overwritten (see copy(), defaults to false)
-     *                                     - $options['copy_on_windows'] Whether to copy files instead of links on Windows (see symlink(), defaults to false)
-     *                                     - $options['delete'] Whether to delete files that are not in the source directory (defaults to false)
+     * @param string            $originDir      The origin directory
+     * @param string            $targetDir      The target directory
+     * @param \Traversable|null $originIterator Iterator that filters which files and directories to copy, if null a recursive iterator is created
```

Comment:
I really hate that PHP have made parameter names part of the BC contract with "named parameters". 

Symfony does not include that in the BC promise, however I dont think we should rename parameters just for fun =)
Will this PR be much worse of if we didnt rename this variable?

Reply:
We certainly don't _have_ to rename anything, I just think it would make the distinction between the two iterators more obvious.

Is renaming a parameter likely to break anything downstream? I would have thought not. The goal here was to preserve backwards compatibility (otherwise I would have also changed the parameter order).

Reply:
if downstream uses named arguments for the call, the renaming is a BC break